content/docs/dvclive/live/log_plot.md

```diff
@@ -0,0 +1,84 @@
+# Live.log_plot()
+
+```py
+def log_plot(
+    self,
+    name: str,
+    datapoints: List[Dict],
+    x: str,
+    y: str,
+    template: Optional[str] = None,
+    title: Optional[str] = None,
+    x_label: Optional[str] = None,
+    y_label: Optional[str] = None,
+):
+```
+
+## Usage
+
+```py
+datapoints = [{"foo": 1, "bar": 2}, {"foo": 3, "bar": 4}]
+
+with Live() as live:
+    live.log_plot(
+        "foo_default", datapoints, x="foo", y="bar", title="Default Linear Plot")
+
+    live.log_plot(
+        "foo_scatter",
+        datapoints,
+        x="foo",
+        y="bar",
+        template="scatter",
+        title="Plot using Scatter template",
+        y_label="CUSTOM LABEL: BAR",
+        x_label="CUSTOM LABEL: FOO"
+    )
```

Comment:
If we include a description of the defaults below, I don't think we need a default example. 

Maybe we can have one example but make it slightly more realistic? Something like https://dvc.org/doc/command-reference/plots/show#example-horizontal-bar-plot or https://dvc.org/doc/command-reference/plots/show#example-scatter-plot might be good.

Reply:
Added the horizontal bar because the full datapoints can be shown in the example without taking too much space, so the snippet is 100% copy-pasteable.